web-api/openapi/back_office.yaml

```diff
@@ -4,20 +4,39 @@ info:
   version: 1.0.0-rc6
   description: |-
     # Back Office
-    For this system, a document is anything that has an id, a snippet, and an arbitrary set of properties.
-    The id uniquely identifies the document in the system.
-    The system uses the snippet to create a document representation that can be matched against user interests.
+    The back office is typically used within server-side apps.
+    This SDK acts as a create/read/update/delete interface for anything related to documents.
 
-    It is possible to attach any properties to a document; the system will not use them but will be returned together with the document when asking for personalized documents for a user. They could help store information like a title and the URL where it is possible to find the document. Each document can have its own set of properties.
+    ## Document
+    By document, we refer to a cohesive text, for example a complete news article.
+    However, we only require a simple representation for our system to work. Consisting just of a unique ID, a text snippet and optional properties.
+    The text snippet is ideally a short, meaningful representation of the larger document, reduced to just one or two paragraphs.
 
-    The API offers an endpoint that allows uploading the documents to the system. After a successful upload, it will be possible to use the document in other endpoints.
+    ### Id
+    The document_id is a unique identifier for a single document.
+    We don't require a specific format, but would suggest using UUID for example.
```

Comment:
We don't require a specific format; it must comply with the following regexp `^[a-zA-Z0-9_:@.-]+$`.

Reply:
Also, it is max 256 long and cannot be empty.
This is the specification https://github.com/xaynetwork/xayn_discovery_engine/blob/main/web-api/openapi/schemas/id.yml

Reply:
The limit is not technical so if needed can be increased. Is to have some bound and avoid ids that are too long because it can hurt performance.

Reply:
Added the regexp before I saw the other comments, I guess it's fine, can also mention the limit in the email